hivpy.yaml

```diff
@@ -0,0 +1,12 @@
+EXPERIMENT: 
+  population: 1000
+  start_year: 1989
+  end_year: 2021
+  time_interval_days: 90
+  simulation_output_dir: output
```

Comment:
Looking back at [the issue comment](https://github.com/UCL/hivpy/issues/31#issuecomment-1131793090), I'd noted that we could move this out of this section and instead have a general "Output" section with the logging and other output options. I don't see any problem doing it this way either (probably better for consistency too), just writing it down so there's a record if we look back in the future!